### PR TITLE
Remove Publications

### PR DESCRIPTION
**Pull request by bmbouter**
pulp_ansible now expects users to distribute content directly from
RepositoryVersion objects by setting the
`Distribution.repository_version` attribute.

Required PR: https://github.com/pulp/pulpcore/pull/118

https://pulp.plan.io/issues/4765
closes #4765

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -256,25 +256,39 @@ Add content to repository ``foo``
...
     }
 
-``$ export PUBLICATION_HREF=$(http :24817/pulp/api/v3/publications/ansible/ansible/ | jq -r '.results[0] | ._href')``
 
 
-Create a Distribution for the Publication
------------------------------------------
+Create a Distribution for a RepositoryVersion
+---------------------------------------------
+
```
Comment (asmacdo) [inline README.rst]: Not sure how you prefer to do this, but in one of the PRs I suggest adding a link to 
https://github.com/pulp/pulpcore/pull/118/files#diff-9e0bbefde3f80fb5c48539015131cfefR38
https://github.com/pulp/pulpcore/pull/118/files#diff-9e0bbefde3f80fb5c48539015131cfefR64

**Comment by daviddavis** [comment]
I tried testing this PR out against pulpcore master but hit some problems. It looks like it needs to be updated to account for the changes in https://github.com/pulp/pulpcore/pull/124.

**Comment by daviddavis** [comment]
I opened a new PR that includes this change plus the fix for https://pulp.plan.io/issues/4820

https://github.com/pulp/pulp_ansible/pull/98

**Comment by bmbouter** [comment]
I'm closing this PR and testing the new PR from @daviddavis here: https://github.com/pulp/pulp_ansible/pull/98

**Pull request closed without merging**